ClosedXML/Excel/CalcEngine/Functions/MathTrig.cs

```diff
@@ -927,8 +928,22 @@ private static object SqrtPi(List<Expression> p)
 
         private static object Subtotal(List<Expression> p)
         {
+            var expressions = p.Skip(1)
+                .SelectMany(e =>
+                    e is XObjectExpression xoe && xoe.Value is CellRangeReference crr
+                        ? crr.Range.CellsUsed()
+                            .Where(z =>
+                            {
+                                if (!z.HasFormula) return true;
+                                var skipedText = Regex.Replace(z.FormulaA1, @"(""([^""]|"""")*"")", "", RegexOptions.Compiled); // skip text in formula
```

Comment:
To avoid instantiation on every iteration, make this regex a `private static readonly` field, similar to the `Regex` fields in `XLCell`.

@Pankraty Agree?